python/lsst/galaxy_shear/runShear.py

```diff
@@ -0,0 +1,293 @@
+import argparse
+import os
+import shutil
+import signal
+import sys
+import time
+import subprocess
+
+
+import lsst.pex.config as pexConfig
+import lsst.afw.table
+
+from great3sims import constants, run
+from lsst.galaxy_shear.shearConfig import RunShearConfig
+from lsst.galaxy_shear.analyzeShearTest import runAnal
+
+#    input a list of pids which we want to wait on
+#    return when all are done, and none has returned an error
```

Comment:
This comment should be a docstring (and go just below the function `def` statement).